modules/quality/test/test_mae.cpp

```diff
@@ -0,0 +1,66 @@
+#include "test_precomp.hpp"
+
+#define TEST_CASE_NAME CV_Quality_MAE
+
+namespace opencv_test
+{
+namespace quality_test
+{
+
+const cv::Scalar
```

Comment:
static? to avoid symbols exporting

Reply:
I made it static, but I was thinking perhaps it can go in the `test_precomp.hpp` in the same place as `MSE_EXPECTED_1` and `MSE_EXPECTED_2`?